src/expr/src/scalar/func.rs

```diff
@@ -2964,6 +2965,36 @@ fn jsonb_build_object<'a>(datums: &[Datum<'a>], temp_storage: &'a RowArena) -> D
     }
 }
 
+fn array_create_multidim<'a>(
```

Comment:
I think these functions could use comments, since their parameters aren't named it was sort of hard for me to figure out how they work.

Reply:
Sure thing! Done.